v23.1/cursors.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Cursors
+summary: Paginate results from queries against your cluster using SQL cursors
+toc: true
+---
+
+A cursor is a placeholder into a selection query that allows you to iterate over subsets of the rows returned by that query.
+
+Cursors differ from [keyset pagination](pagination.html) and [`LIMIT`/`OFFSET`](limit-offset.html) in that:
+
+- Each cursor is a stateful SQL object that is referred to by a unique name.
+- Each cursor requires holding open its own dedicated (read-only) [transaction](transactions.html).
+- Each cursor operates on a snapshot of the database at the moment that cursor is opened.
+
+## Synopsis
+
+Cursors are declared and used with the following keywords:
+
+- [`DECLARE`](sql-grammar.html#declare_cursor_stmt)
+- [`FETCH`](sql-grammar.html#fetch_cursor_stmt)
+- [`CLOSE`](sql-grammar.html#close_cursor_stmt)
+
+<div>
```

Comment:
I think the bogus Vale failures are caused by the `remote_include` being surrounded by the `<div>` tag.

Reply:
Interesting, I haven't seen that before.  AFAICT we always wrap the SVG diagrams in the `div` tags everywhere else.  Not sure why it's mad here.  Oh well hope the next one passes? :-)